### PR TITLE
suggest `wrangler init` if wrangler.toml does not exist

### LINKED ISSUE #827: if no wrangler.toml found on build/preview/publish, give an error message to run wrangler init

### PR DESCRIPTION
**Pull request by ashleymichal**
closes #827

### CODE REVIEW

File: src/settings/toml/manifest.rs [hunk 1]
```diff
@@ -46,6 +46,10 @@ pub struct Manifest {
 
 impl Manifest {
     pub fn new(config_path: &Path) -> Result<Self, failure::Error> {
+        failure::ensure!(
+            config_path.exists(),
+            "wrangler.toml not found; run `wrangler init` to create one."
```
Comment (EverlastingBugstopper) [inline src/settings/toml/manifest.rs:51]: nit: add link to docs about how to configure

File: src/settings/toml/tests/mod.rs [hunk 2]
```diff
@@ -147,5 +147,5 @@ fn base_fixture_path() -> PathBuf {
 }
 
 fn toml_fixture_path(fixture: &str) -> PathBuf {
-    base_fixture_path().join(fixture)
+    base_fixture_path().join(fixture).with_extension("toml")
```
Comment (EverlastingBugstopper) [inline src/settings/toml/tests/mod.rs:150]: why is this change necessary?

Comment (ashleymichal) [reply src/settings/toml/tests/mod.rs:150]: some magical thing in `config` crate was doing this for us when we went to use `merge`, but now that we first check `config_path.exists()` before we get to that point, the extension needs to be explicitly included in the path.
